schema/gene.js

```diff
@@ -27,119 +35,145 @@ export const GeneType = new GraphQLObjectType({
-            return Object.assign(
-              { aggregations }, // Add data to connection so the `aggregations` connection field can resolve it
-              connectionFromArraySlice(hits, options, {
-                arrayLength: aggregations.total.value,
-                sliceStart: gravityOptions.offset,
-              })
-            )
-          })
       },
-    },
-    description: {
-      type: GraphQLString,
-    },
-    display_name: {
-      type: GraphQLString,
-    },
-    filtered_artworks: filterArtworks("gene_id"),
-    href: {
-      type: GraphQLString,
-      resolve: ({ id }) => `gene/${id}`,
-    },
-    image: Image,
-    is_published: {
-      type: GraphQLBoolean,
-      resolve: ({ published }) => published,
-    },
-    mode: {
-      type: GraphQLString,
-      resolve: ({ type }) => {
-        const isSubjectMatter = type && type.name && type.name.match(SUBJECT_MATTER_REGEX)
-        return isSubjectMatter ? "artworks" : "artist"
+      description: {
+        type: GraphQLString,
       },
-    },
-    name: {
-      type: GraphQLString,
-    },
-    trending_artists: {
-      type: new GraphQLList(Artist.type),
-      args: {
-        sample: {
-          type: GraphQLInt,
+      display_name: {
+        type: GraphQLString,
+      },
+      filtered_artworks: filterArtworks("gene_id"),
+      href: {
+        type: GraphQLString,
+        resolve: ({ id }) => `gene/${id}`,
+      },
+      image: Image,
+      is_published: {
+        type: GraphQLBoolean,
+        resolve: ({ published }) => published,
+      },
+      mode: {
+        type: GraphQLString,
+        resolve: ({ type }) => {
+          const isSubjectMatter = type && type.name && type.name.match(SUBJECT_MATTER_REGEX)
+          return isSubjectMatter ? "artworks" : "artist"
         },
       },
-      resolve: ({ id }, options) => {
-        return gravity(`artists/trending`, {
-          gene: id,
-        }).then(artists => {
-          if (_.has(options, "sample")) return _.take(_.shuffle(artists), options.sample)
-          return artists
-        })
+      name: {
+        type: GraphQLString,
       },
-    },
+      similar: {
+        type: geneConnection, // eslint-disable-line no-use-before-define
+        args: pageable({
+          exclude_gene_ids: {
+            type: new GraphQLList(GraphQLString),
+            description: "Array of gene ids (not slugs) to exclude, may result in all genes being excluded.",
+          },
+        }),
+        description: "A list of genes similar to the specified gene",
+        resolve: (gene, options, request, { rootValue: { similarGeneLoader } }) => {
+          const { limit: size, offset } = getPagingParameters(options)
+          const gravityArgs = { size, offset, exclude_gene_ids: options.exclude_gene_ids, total_count: true }
+
+          return similarGeneLoader(gene.id, gravityArgs).then(({ body, headers }) => {
+            const genes = body
```

Comment:
you probably don't need this two `const` and can just pass it in directly to `connectionFromArraySlice` #minor

Reply:
we added the two different `const` to make it a little more explicit.